frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -204,7 +198,7 @@ export default function PatientsPage({
   const sampleQueryParamFieldName = "patientId";
   const sampleQueryParamValue = params[sampleQueryParamFieldName];
   const defaultSort = {
-    colId: "dmpPatientId",
+    colId: "latestImportDate",
     sort: AgGridSortDirection.Desc,
   };
```

Comment:
Updated Patients page's default sort to use `importDate` for card [#1321](https://app.zenhub.com/workspaces/smile-scrum-5fcfa28a4ccc1600165347b6/issues/gh/mskcc/smile-server/1321)